parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -122,6 +134,14 @@ ensureRPC() {
     systemctlEnableAndStart rpc-statd || exit $ERR_SYSTEMCTL_START_FAIL
 }
 
+ensureAuditD() {
+	if [[ "${AUDITD_ENABLED}" == true ]]; then
+		systemctlEnableAndStart auditd || exit $ERR_SYSTEMCTL_START_FAIL
+	else
+	  apt list --installed | grep 'auditd' && systemctlDisableAndStop auditd || exit $ERR_SYSTEMCTL_START_FAIL
```

Comment:
should we uninstall it?

Reply:
I considered that but there should be no functional downside of just keeping it there, disabled. The downside of removing it is the add'l apt operational dependency in CSE.